types/node/test.d.ts

```diff
@@ -391,32 +391,32 @@ declare module "node:test" {
     class TestContext {
         /**
          * This function is used to create a hook running before subtest of the current test.
-         * @param fn The hook function. If the hook uses callbacks, the callback function is passed as
-         *    the second argument. Default: A no-op function.
+         * @param [fn='A no-op function'] The hook function. If the hook uses callbacks, the callback function is passed as
```

Comment:
A quick check seems to show that this style actually makes the description of what the default is disappear. I would revert this style change. https://www.typescriptlang.org/play?#code/MYGwhgzhAEAqCmEAuBhA9gOyfAHk6A3gFDSlmkD0AVFSeeVXABYCWMAZgK4bBIubQ20ThHgATaEjTRgAJ3hhs0MNCZo0Aa2izuGFhgDm0AEbx2aedAidj2ZNDTtJTeDM6z5WSYiQA6OvSkjAACAA5gsmAAttAA2uwYALwA5ACC0BhoALRoodBcPHyYyQC6zK5qmvncvPwYvtAAkk5ILqrqWiKIMmAgIMZgwBoQADTOrsC9-YNaBbUCQuFQ4soQAYGMZK2uosCYEhEGnFHwWP6BZCHhkTG5RRgw6BjsLEeR9w6h9xwW4+2a5wu0BCEH0wFcADcAEwABl8AEZfDD1vQqBQUWRTOZ5ABCABc0AAFAkCWAMABPACU0ESAD5lBSANxEAC+RCIGHgAHc4D4ntg8ITKb4sRZ4EKgA